content/showcase/_index.html

```diff
@@ -12,10 +12,18 @@ <h1 class="title is-2">UQCS Showcase</h1>
         <div class="columns is-centered">
             <div class="column is-8">
                 <div class="showcase-gallery columns is-multiline">
+                    {{< showcase-item
+                        src="/img/media/showcase/codeduel.png"
+                        title="CodeDuel"
+                        caption="<p> Bored of Leetcoding by yourself? Well, challenge a friend to see who's better! <br> Oh, did we mention you can <b> sabotage </b> them and watch them lose their minds? 🤯 </p>"
```

Comment:
To try to be consistent with the other showcase items, it might be nice to remove the `<br>`.